script/v2/keeper-actions/RebalanceScUsdcV2.s.sol

```diff
@@ -120,6 +120,11 @@ contract RebalanceScUsdcV2 is ScUsdcV2ScriptBase {
                 targetLtv: aaveV3TargetLtv
             })
         );
+
+        require(
+            morphoInvestableAmountPercent + aaveV2InvestableAmountPercent + aaveV3InvestableAmountPercent == 1e18,
```

Comment:
This will cause error right if an adapter is not supported and its target ltv is zero?

Reply:
it doesn't make sense for us to set investableAmountPct != 0 and targetLtv = 0 for the same adapter, right?

Reply:
[fixed](https://github.com/lindy-labs/sandclock-contracts/pull/132/commits/f24042330f47595621bdad4f1ce858a3630ad420)